lib/packwerk/package_set.rb

```diff
@@ -12,7 +12,7 @@ class PackageSet
     extend T::Generic
     include Enumerable
 
-    Elem = type_member(fixed: Package)
+    Elem = type_member { { fixed: Package } }
```

Comment:
Ruby syntax strikes again... This is a block returning a hash. It took me a second to realize that.